qa/L0_model_config/autofill_noplatform/custom/unknown_backend.unknown/expected

```diff
@@ -1 +1 @@
-Invalid argument: unable to find 'libtriton_unknown.so' for model 'unknown_backend.unknown'
+Invalid argument: unable to find 'libtriton_unknown.so' or 'unknown/model.py' for model 'unknown_backend.unknown'
```

Comment:
nit: space

Reply:
Is this in reference to the lack of a newline or is there a space issue I am not seeing here?

Reply:
The lack of newline. Sorry for being unclear

Reply:
No worries! I didn't use a newline because the previous version did not have one and figured the comparison in bash would be sensitive to this change. Taking a closer look in the repo it seems like we are inconsistent in our use of a newline. I know our bash files should end with one. Do we try to apply this principle to all files?

Reply:
The `newline` check should have fixed this. I think it just doesn't check for this sort of file. I saw the previous file does have an extra line in it but your change deleted that. I think it would be good to change that though (although not a urgent or strong preference)

Reply:
I'll make a note to include this in my next server-based PR that way I don't have to create and new PR just for this.